src/kimmdy/coordinates.py

```diff
@@ -656,7 +656,7 @@ def merge_pairs(break_pair: Optional[Pair], bind_pair: Optional[Pair]):
                     ff.improper_dihedraltypes,
                     molA,
                     molB,
-                    "9",
+                    "4",
```

Comment:
Should we turn these magic numbers into constants to make them easier to spot?

Reply:
yes, good point, and it was worth it, few lines above the same typo was still there.